### PR TITLE
Add detekt ruleset to team-props

### PR DESCRIPTION
**Pull request by tobiasheine**
Tracked by [PT-273](https://novoda.atlassian.net/browse/PT-273)

## This PR adds

The default rules for [detekt](https://github.com/arturbosch/detekt) to be used for new projects.

### CODE REVIEW

**Comment by Mecharyry** [comment]
Worth adding anything to the readme? https://github.com/novoda/novoda/blob/master/team-props/README.md

**Comment by tobiasheine** [comment]
Good point, missed that. Just saw the readme in the root folder ^^

File: team-props/README.md [hunk 1]
```diff
@@ -55,19 +55,19 @@ It currently does _not_ support Kotlin static analysis such as KtLint and Detekt
...
- 6. Configure the static analysis settings from the `team-props/static-analysis.gradle` file
- 7. Add the following Lint configuration to the `build.gradle` of all your Android projects:
+ 7. Configure the static analysis settings from the `team-props/static-analysis.gradle` file.
+ 8. Configure detekt (optional)
+
+    In case you would like to use [detekt](https://github.com/arturbosch/detekt) to analyse your kotlin code, add the following section to the `team-props/static-analysis.gradle` file:
+
     ```gradle
-    //...
-    android {
```
Comment (tobiasheine) [inline team-props/README.md]: I guess we don't need that anymore since lint is configured as part of the plugin.

**Comment by tobiasheine** [comment]
@rock3r I updated the PR 👍

**Comment by rock3r** [comment]
@Mecharyry all yours

**Pull request merged**